app/components/metrics-counter.js

```diff
@@ -0,0 +1,20 @@
+import Component from '@ember/component';
+import { pluralize } from 'ember-inflector';
+
+export default Component.extend({
+  tagName: 'div',
+  classNames: [ 'col-lg-3', 'col-md-4' ],
+  data: null,
+  label: '',
+  count: 0,
+  init() {
+    this._super();
+  },
+
+  didReceiveAttrs() {
```

Comment:
Why are you not doing this in the template, using the pluralize helper?

Reply:
Because it's not a label which is pluralized but a key value